client/src/actions/authActions.js

```diff
@@ -1,41 +1,63 @@
 import axios from 'axios';
-import { SET_CURRENT_USER, UNSET_CURRENT_USER } from '../actions/types';
+import { SET_CURRENT_USER, UNSET_CURRENT_USER, SET_EXISTENCE }
+  from '../actions/types';
 import setAuthorizationHeader from '../utils/setAuthorizationToken';
 
 /**
  * Set Current User
+ *
  * @description Set the current user
+ *
  * @param {object} user - user to set
+ *
  * @returns {object} action
  */
 const setCurrentUser = user =>
   ({ type: SET_CURRENT_USER, user, });
```

Comment:
i noticed you left a comma after `user` and there is no other value after it

Reply:
Yes, Thanks fixed this too. Looking forward to more of your review.